proj/src/sub1/test/a.ml

```diff
@@ -2,10 +2,12 @@
    Tests for Sub1.A
 *)
 
+let check msg x = Alcotest.(check bool) msg true x
+
 let test_time () =
-  assert (Proj_sub1.A.now () > 1000.);
-  assert (Proj_sub1.A.now () > 1_522_882_648.)
+  check "now is greater than 1000" (Proj_sub1.A.now () > 1000.);
+  check "now is fix" (Proj_sub1.A.now () > 1_522_882_648.)
```

Comment:
I know people like to do that, but I don't like to have to provide a string that paraphrases the code for each assertion. The test itself has a name, so that should give the user a quick sense of what's wrong before they dive into the source code.